src/energyplus/ForwardTranslator/ForwardTranslateAirLoopHVACOutdoorAirSystem.cpp

```diff
@@ -117,12 +100,34 @@ namespace energyplus {
       }
     }
 
-    for (auto& controller : controllers) {
-      auto _controller = translateAndMapModelObject(controller);
-      if (_controller) {
+    // If it's not empty: then we have work to do.
+    // If it's empty, there's at least the controller Outdoor Air UNLESS it's on a AirLoopHVACDOAS in which case do nothing
+    if (!controllers.empty() || !modelObject.airLoopHVACDedicatedOutdoorAirSystem()) {
+
+      IdfObject _controllerList(IddObjectType::AirLoopHVAC_ControllerList);
```

Comment:
This is the actual fix. The ControllerList is translated only if we have work to do. This is based on how I understand E+ expects it...